app/views/microposts/_micropost.html.erb

```diff
@@ -0,0 +1,13 @@
+<li id="micropost-<%= micropost.id %>">
+  <%= link_to gravatar_for(micropost.user, size: Settings.size_user), micropost.user %>
+  <span class="user"><%= link_to micropost.user.name, micropost.user %></span>
+  <span class="content"><%= micropost.content %></span>
+  <%= image_tag micropost.picture.url if micropost.picture? %>
+  <span class="timestamp">
+  <%= t("posted") %><%= time_ago_in_words(micropost.created_at) %> ago.
```

Comment:
Nó sẽ lấy thời điểm hiện tại trừ đi thời gian create để tính ra khoảng thời gian để in ra màn hình, ví dụ :"5 day ago"